Gemfile

```diff
@@ -14,7 +14,7 @@ gem 'honeybadger' # for error reporting / tracking / notifications
 group :development, :test do
   gem 'pry-byebug'
   gem 'rubocop', '~> 0.52.1', require: false # avoid code churn due to rubocop changes
-  gem 'rubocop-rspec'
+  gem 'rubocop-rspec', '~> 1.23.0', require: false # avoid code churn due to rubocop-rspec changes
```

Comment:
Don't think we can legitimately do `require: false` on this or the preceeding line AND expect CI to run rubocop.